WordPress/Classes/ViewRelated/Pages/PageListViewController.swift

```diff
@@ -45,6 +45,20 @@ class PageListViewController: AbstractPostListViewController, UIViewControllerRe
         }
     }
 
+    @objc private lazy var _syncHelper: PageContentSyncHelper = {
+        let syncHelper = PageContentSyncHelper()
```

Comment:
I think the change I suggested above would avoid the need for this or for customizing the getter/setter below.